src/authentication/footer/footer.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import { bem } from '../../lib/bem';
+import './styles.scss';
+
+const c = bem('invite-footer');
+
+export class Footer extends React.Component {
+  render() {
+    return (
+      <div className={c('')}>
+        <div>
+          <span>Already on ZERO? </span>
+          <Link to='/login'>Log in</Link>
+        </div>
+
+        <div>
+          <span>Dont have an invite code? </span>
```

Comment:
```suggestion
          <span>Don't have an invite code? </span>
```

Reply:
I'll check this with Eth as this is what is shown in figma :)